CONST_vars.yaml

```diff
@@ -82,12 +82,11 @@ vars:
     # pyramid_closure configuration
     closure_library_path: 'process.stdout.write(require("openlayers/node_modules/closure-util").getLibraryPath())'
     pyramid_closure:
-        roots:
-        - "{closure_library_path}/closure/goog"
         roots_with_prefix:
-            ../../../../../../../../../proj/js: "{directory}/{package}/static/js"
-            ../../../../../../../../../node_modules/openlayers: "{directory}/node_modules/openlayers"
-            ../../../../../../../../../node_modules/ngeo/src: "{directory}/node_modules/ngeo/src"
+        - "{closure_library_path}/closure/goog {closure_library_path}/closure/goog"
+        - "{package}:static/js {directory}/{package}/static/js"
+        - "{node_modules_path}/openlayers {node_modules_path}/openlayers"
+        - "{node_modules_path}/ngeo/src {node_modules_path}/ngeo/src"
```

Comment:
We speak about having a config like `- ["{node_modules_path}/ngeo/src", "{node_modules_path}/ngeo/src"]` to don't post process the strings ...